Makefile.in

```diff
@@ -80,7 +89,7 @@ EXTRA_PCRE2 = @EXTRA_PCRE2@
 # Various flags
 #
 CXXFLAGS = @CXXFLAGS@ $(EXTRA_CXXFLAGS)
-CPPFLAGS = -DLOCALEDIR=\"$(localedir)\" -DPREFIX=L\"$(prefix)\" -DDATADIR=L\"$(datadir)\" -DSYSCONFDIR=L\"$(sysconfdir)\" -DBINDIR=L\"$(bindir)\" -DDOCDIR=L\"$(docdir)\" -iquote. -iquote./src/
+CPPFLAGS = -D_UNICODE=1 -DLOCALEDIR=\"$(localedir)\" -DPREFIX=L\"$(prefix)\" -DDATADIR=L\"$(datadir)\" -DSYSCONFDIR=L\"$(sysconfdir)\" -DBINDIR=L\"$(bindir)\" -DDOCDIR=L\"$(docdir)\" -iquote. -iquote./src/
```

Comment:
Why would we care about that? We don't support building fish as a native Windows app. And this is the only approved mechanism by the MuParser project for enabling wide character support.